lab2/XYHVPath path
string path_name
---
bool success
